server/src/domain/system-config/dto/system-config-oauth.dto.ts

```diff
@@ -23,6 +23,11 @@ export class SystemConfigOAuthDto {
   @IsString()
   clientSecret!: string;
 
+  @ValidateIf(isEnabled)
+  @IsNotEmpty()
+  @IsNumber()
+  defaultStorageQuota!: number;
```

Comment:
This probably doesn't belong in the oauth settings section.

Can we move it to it's own section? Maybe it can be under a new one called "user settings".

Reply:
Hmm, I was only attempting to set a default quota for oauth registered users if the did not have a claim. If this quota becomes global, should the create user box be populated with this default value and allow the user to manually change it?

Reply:
Oh actually, this might be fine. When you create a new user in the web you have an option to specify a quota at that time, so the default probably would not ever be used. I am ok leaving this as an OAuth only setting.